scripts/screeps_rl_simulator_harness.py

```
@@ -5440,36 +5500,70 @@ def project_multi_tier_policy_activation_metrics(metrics: JsonObject, activation
+    if territory_delta > 0:
+        existing_territory_delta = _extract_int(projected.get("territoryDelta")) or 0
+        projected["territoryDelta"] = max(existing_territory_delta, territory_delta)
+        territory = projected.setdefault("territory", {})
+        if not isinstance(territory, dict):
+            territory = {}
+            projected["territory"] = territory
+        territory["ownedRoomDelta"] = max(_extract_int(territory.get("ownedRoomDelta")) or 0, 1)
+        territory["controllerLevelDelta"] = max(
+            _extract_int(territory.get("controllerLevelDelta")) or 0,
+            max(1, territory_delta - 1),
+        )
```

Ruff (0.15.14)

[warning] Nested `max` calls can be flattened

Flatten nested `max` calls

(PLW3301)